lld/test/ELF/riscv-vendor-relocations.s

```diff
@@ -0,0 +1,28 @@
+# RUN: llvm-mc -triple riscv32 -mattr=+experimental-xqcibi,+xandesperf %s -filetype=obj -o %t.o
+# RUN: not ld.lld -pie %t.o -o /dev/null 2>&1 | FileCheck %s
+
+  .option exact
+
+  qc.e.bgeui s0, 20, TARGET
+# CHECK: unknown vendor-specific relocation (193) in vendor namespace "QUALCOMM" against symbol TARGET
```

Comment:
In check prefixes we always inspect `error: ` for errors (most tests follow this convention)